include/swift/AST/DiagnosticsParse.def

```diff
@@ -216,6 +216,8 @@ ERROR(expected_decl,none,
       "expected declaration", ())
 ERROR(expected_identifier_in_decl,none,
       "expected identifier in %0 declaration", (StringRef))
+ERROR(expected_keyword_in_decl,none,
+      "expected '%0' keyword in %1 declaration", (StringRef, StringRef))
```

Comment:
Nitpick: you can use DescriptiveDeclKind for the second parameter here.

Reply:
Okay, will do

Reply:
@jrose-apple can you please review and trigger CI again?